code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -40,6 +41,7 @@
 /datum/reagent/drug/nicotine
 	name = "Nicotine"
 	id = "nicotine"
+	value = 1
```

Comment:
0.1
Its a bit better then just no vaule and it stops the issue. As well as makes tobacco plants still a viable cash crop

Reply:
idk if byond likes tiny number